packages/testbed/src/App.js

```diff
@@ -0,0 +1,29 @@
+import './App.css'
+
+import React, {Component} from 'react'
+
+import { Blaster, Button, Box } from "@blasterjs/core";
+
+const theme = {
```

Comment:
as soon as you try to override base styles our callstack error comes back.

Try adding the following to reproduce the error.
```
colors: {
    primary: "red"
},
button: {
    ...
}
```

Reply:
I see. Good catch.